hydromt_wflow/workflows/soilparams.py

```diff
@@ -0,0 +1,53 @@
+"""Functions for individual soil parameters."""
+
+import numpy as np
+import xarray as xr
+
+
+def ksathorfrac(
+    da: xr.DataArray,
+    ds_like: xr.Dataset,
+    resampling_method: str,
+) -> xr.Dataset:
```

Comment:
Still think you should return a DataArray

Reply:
No It should return a Dataset

Reply:
Yes a DataArray can be passed and you can even change the name when doing so:
``` self.set_grid(da, name="KsatHorFrac") ```

Reply:
Allright, then that seems better!